regression/apps/fork/fork.s

```diff
@@ -22,13 +22,10 @@ _child:
     call    exit
 wait_child:
```

Comment:
After the modification, the `wait_child` label is no longer appropriate.

Reply:
The original description of this commit is wrong. We remove the option `WNOHANG`, instead of `WNOWAIT`. So the label should still be `wait_child`.